src/endless/Analytics.cpp

```diff
@@ -331,6 +332,7 @@ void Analytics::urlEncode(const CString &in, CString &out)
 
 void Analytics::prefixId(CString &id)
 {
-	id.Format(_T("v=1&tid=%s&cid=%s&an=Endless%%20Installer&av=%s&ul=%s&cd1=%s&"),
-		m_trackingId, m_clientId, _T(RELEASE_VER_STR), m_language, m_windowsVersion);
+	id.Format(_T("v=1&tid=%s&cid=%s&an=Endless%%20Installer&av=%s&ul=%s&cd1=%s&cd2=%s"),
+		m_trackingId, m_clientId, _T(RELEASE_VER_STR), m_language, m_windowsVersion,
+		CEndlessUsbToolDlg::IsCoding() ? "code" : "eos");
```

Comment:
Right, I missed the fact that more parameters were being added. Also fixed in last push.